src/tree.js

```diff
@@ -56,7 +56,7 @@ AvlTree.prototype.insert = function (element) {
 //TODO: allow duplicates
 AvlTree.prototype._insert = function (element, node) {
 	if (node === null) {
-		return new Node(element);
+		return new TreeNode(element);
```

Comment:
Node was renamed to TreeNode due to it having some sort of conflict with Bootstrap dynamic pills